### DISCUSSION TITLE
Partial regeneration

### DISCUSSION THREAD

**Discussion by dmudro** [vercel/next.js#27947] [Ideas]
We are happy Vercel customers and (eraly) Next.js developers. There are a few areas / features in Next.js in which I've been waiting for progress for quite some time though.

Check out this [version 9.4 announcement ](https://nextjs.org/blog/next-9-4#incremental-static-regeneration-beta) from May 2020:

> Up next, we will be working on a supplemental RFC to address two additional incremental static generation capabilities:
>
> - Re-generating and invalidating multiple pages at once (like your blog index and a certain blog post)
> - Re-generating by listening to events (like CMS webhooks), ahead of user traffic

Partial regeneration on demand sounds awesome. Is it still planned or has it been delivered already?

**Comment by martinzco** [comment]
@leerob commented a while ago here: https://github.com/vercel/next.js/discussions/11552#discussioncomment-989514

**Reply by leerob** [reply]
Correct, we are still working on this!
